docs/features/logforwarding/sources/http-source.adoc

```diff
@@ -0,0 +1,23 @@
+=== Receiving logs over HTTP
```

Comment:
@alanconway it looks like this never actually made it into the docs at https://github.com/openshift/cluster-logging-operator/tree/master/docs/features/logforwarding but seems like it would be good source material for https://issues.redhat.com/browse/OBSDOCS-212 - is the content here still accurate?